core/src/main/java/org/infinispan/interceptors/distribution/BaseDistributionInterceptor.java

```diff
@@ -358,44 +360,55 @@ public Object visitGetAllCommand(InvocationContext ctx, GetAllCommand command) t
          }
          return invokeNext(ctx, command);
       }
+      GetAllSuccessHandler getAllSuccessHandler = new GetAllSuccessHandler(command);
```

Comment:
Yes, I need a way to fetch multiple entries (`remoteGetAll`) from `TxDistributionInterceptor`, because when xsite is on, we will need the final value of the entry to replicate xsite.